std/experimental/allocator/mallocator.d

```diff
@@ -198,7 +212,8 @@ struct AlignedMallocator
     /**
     Forwards to $(D alignedAllocate(bytes, platformAlignment)).
     */
-    @nogc @trusted void[] allocate(size_t bytes) shared
+    @trusted @nogc nothrow
```

Comment:
> When I was writing this, I knew that your MMM PR was rejected (I didn't remember why), so I decided to leave this attribute as it is, in order to make it non-controversial for merging.

Nevermind, I didn't see that `@trusted` was already there. It is misused - `allocate` is provably memory safe.